crates/sui-replay/src/lib.rs

```diff
@@ -584,6 +608,26 @@ pub async fn execute_replay_command(
     })
 }
 
+fn get_filepath(tx_digest: TransactionDigest, extension: &str) -> PathBuf {
+    let mut filename = OsString::new();
+    filename.push("replay_output_");
+    filename.push(tx_digest.to_string());
+    filename.push(".");
+    filename.push(extension);
+    let mut output_path = std::path::PathBuf::from(".");
```

Comment:
maybe there is a specific reason here to get cwd like this, but probably this is a bit better, up to you.
```suggestion
    let mut output_path = std::env::current_dir().map_err(|_| anyhow!("Cannot get current working directory")?;
```